package.json

```diff
@@ -73,7 +73,7 @@
     "./packages/*"
   ],
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=7.0.0 < 10.0.0"
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0 < 10.0.0"
```

Comment:
Why are we changing the npm version?

Reply:
good call `Now using node v16.0.0 (npm v7.10.0)`.  I was thinking of the latest 16 which defaults to 8

Reply:
So are we going to leave the npm versioning the same or....

Reply:
sorry I thought it updated.  On it.